docs/deployment.md

```diff
@@ -47,18 +47,21 @@ environments.
 
 ### Deploying to Staging Directly
 
-1. Add git remote add `git remote add staging git@beta.aptible.com:vita-min-staging/vita-min-staging.git`
-2. If you have not set up ssh to aptible, add your key via [Aptible/ssh keys](https://dashboard.aptible.com/settings/protected/ssh) and paste your ssh `pbcopy < $HOME/.ssh/id_rsa.pub` (or other *.pub key)
-3. Push to staging which triggers a deploy, must push to main or won't deploy, use force flag if staging is not in sync with your branch `git push staging HEAD:main -f`
-
-At times, you will want to push code to Demo or Staging without going
-through the automated deploy process. Manual deployment should be limited
-to situations where a temporary solution is called for.
-
-Good candidates:
-
-- pushing a spike for review by stakeholders
-- manually testing internals or integration with a 3rd party (e.g. Sentry)
+Deploying to staging is the only way to test XML changes since the schemas aren't on Heroku.
+Our current process is to open a PR off the staging branch and merge it so Product
+can perform acceptance tests on this environment.
+
+A rough step by step process will look like:
+1. `git checkout staging`
+2. `git pull`
+3. `git checkout -b {my_new_feature#pivotal-hash1234}`
+4. `git cherry-pick {shah}` # This will be the shah for the commit on main if you have a PR there.  You can also base your work entirely off staging and cherry to main once it's been accepted.
+5. `git push`
+6. Open a PR based on staging branch
+7. Deploy the PR which will kick off the pipeline and deploy to Aptible staging
+
+Manual deployment currently don't seem to be working and is not urgent to fix,
```

Comment:
i'm not sure if it's currently not working or if just my SSH keys just aren't set up correctly so maybe we can omit this line unless we know everyone is having issues

Reply:
It wasn't working for me either, but maybe good to leave out.